data/data/powervs/variables-powervs.tf

```diff
@@ -107,19 +107,19 @@ variable "powervs_sys_type" {
 variable "powervs_base_domain" {
   type        = string
   description = "The base domain name of the cluster"
-  default     = ""
+  default     = "openshift-on-power.com"
```

Comment:
the better thing would be to use the variable from the base config, which is `base_domain` and `cluster_domain` in the dns module.

Reply:
Let's do that in next iteration. All these values should be handled properly without any hard coding

Reply:
agree.